extensions/amp-sidebar/1.0/amp-sidebar-1.0.md

```diff
@@ -58,103 +58,232 @@ limitations under the License.
+  </tr>
+</table>
+
+If the user taps back on the partially-visible main content area, this closes the sidebar.
 
-Alternatively pressing the escape key on the keyboard will also close the sidebar.
+Alternatively, pressing the escape key on the keyboard will also close the sidebar.
 
 Example:
+
 ```html
 <button class="hamburger" on='tap:sidebar1.toggle'></button>
 <button on='tap:sidebar1'>Open</button>
 <button on='tap:sidebar1.open'>Open</button>
 <button on='tap:sidebar1.close'>x</button>
 ```
 
-## Attributes
-
-**side**
+### Toolbar
 
-The `side` attribute may be set to `left` or `right` depending upon whether sidebar should open in the left or right side of the page. If a `side` is not set on the `amp-sidebar` then it will be inherited from the `body` tag's `dir` attribute (`ltr` => `left` , `rtl` => `right`) and if one does not exist then the `side` is defaulted to `left`.
+Toolbar allows duplicating items within the sidebar, into the `<body>`, if the window matches a certain media query
 
-**open**
+#### Behavior
 
-The `open` attribute is present on the sidebar when it is open.
+- The sidebar may implement toolbars by adding nav elements with the toolbar attribute
+- The nav element must be a child of `<amp-sidebar>` and follow this format: `<nav toolbar="(media-query)">`
+    - For instance, this would be a valid use of toolbar: `<nav toolbar="(max-width: 1024px)">`
+- The nav containing the toolbar attribute, must only contain a single `<ul>` element, that contains `<li>` elements
+    - The `<li>` elements may may contain any valid HTML elements (supported by AMP), or any of the AMP elements that `<amp-sidebar>` supports.
+- The nav element, or it's `<ul>`'s `<li>` elements, may also contain the attribute `toolbar-only`
+    - The attribute `toolbar-only` will hide the elements with the attribute in the sidebar, but leave them shown in the toolbar
+- The nav element may also contain the attribute `target` that accepts an element id
+    - The attribute `target` will place the toolbar into the specified id of the element on the page, without the default toolbar styling.
+    - e.g `<nav toolbar="(max-width: 1024px)" target="toolbar-target">`
+- Toolbar behavior is only applied while the media-query is valid
 
-**layout**
-
-The only permissible value for the `layout` attribute in `amp-sidebar` is `nodisplay`.
+Example:
 
-**common attributes**
+```html
+<amp-sidebar id="sidebar1" layout="nodisplay" side="right">
+  <ul>
+    <li> Nav item 1</li>
+    <li> Nav item 2</li>
+    <li> Nav item 3</li>
+    <li> Nav item 4</li>
+    <li> Nav item 5</li>
+    <li> Nav item 6</li>
+  </ul>
 
-This element includes [common attributes](https://www.ampproject.org/docs/reference/common_attributes) extended to AMP components.
+  <nav toolbar="(max-width: 767px)">
+    <ul>
+      <li>
+        <input placeholder="Search..."/>
+      </li>
+    </ul>
+  </nav>
+</amp-sidebar>
+```
 
-## Styling
+Example (toolbar-only on `<nav>` element):
```

Comment:
- Rewrite as "Example: Display only the toolbar (no sidebar)
- Add sentence or two describing the example.